Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+/// Renders a receipt in an AirPrint enabled printer.
+/// To be properly implemented in https://github.com/woocommerce/woocommerce-ios/issues/3978
+final class ReceiptRenderer: UIPrintPageRenderer {
+    private let lines: [ReceiptLineItem]
+    private let paymentIntent: PaymentIntent
+
+    private let attributes: [NSAttributedString.Key: Any] = [.font: UIFont(name: "HelveticaNeue", size: 48) as Any]
+
+    init(content: ReceiptContent) {
+        self.lines = content.lineItems
+        self.paymentIntent = content.paymentIntent
+
+        super.init()
+
+        configureHeaderAndFooter()
+
+        configureFormatter()
+    }
+
+    override func drawHeaderForPage(at pageIndex: Int, in headerRect: CGRect) {
+        let pageNumberString = NSString(string: "Order receipt. Page \(pageIndex + 1)")
+        pageNumberString.draw(in: headerRect, withAttributes: attributes)
+    }
+
+    override func drawContentForPage(at pageIndex: Int, in contentRect: CGRect) {
+        let printOut = NSString(string: "Total charged: \(paymentIntent.amount / 100) \(paymentIntent.currency.uppercased())")
```

Comment:
Final receipts should probably show itemized order list, taxes, etc.?

Reply:
Yep, I would expect so. That was the idea behind passing an array of "LineItem", so that we can provide that information.